### PR TITLE
Update documentation to match proposed update to component

### PR DESCRIPTION
**Pull request by derekatkins**
## Proposed change

This change is coinciding with a PR on the manual_mqtt component.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#40973
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/manual_mqtt.markdown [hunk 1]
```diff
@@ -113,12 +129,24 @@ Additionally, the following MQTT configuration variables are also available.
...
   description: The MQTT topic Home Assistant will publish state updates to.
-  required: true
+  required: false
   type: string
+  default: home/alarm
 command_topic:
   description: The MQTT topic Home Assistant will subscribe to, to receive commands from a remote device to change the alarm state.
-  required: true
+  required: false
+  type: JSON map
```
Comment (klaasnicolaas) [inline source/_integrations/manual_mqtt.markdown]:
```suggestion
  type: string
```

File: source/_integrations/manual_mqtt.markdown [hunk 2]
```diff
@@ -113,12 +129,24 @@ Additionally, the following MQTT configuration variables are also available.
...
+  default: home/alarm/set
+config_topic:
+  description: The MQTT topic Home Assistant will use to publish the alarm configuration for the Alarm Panel to use.
+  required: false
+  type: JSON map
```
Comment (klaasnicolaas) [inline source/_integrations/manual_mqtt.markdown]:
```suggestion
  type: string
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/manual_mqtt.markdown (hunk 1 above)
Comment (derekatkins) [reply source/_integrations/manual_mqtt.markdown]: Thanks

File: source/_integrations/manual_mqtt.markdown (hunk 2 above)
Comment (derekatkins) [reply source/_integrations/manual_mqtt.markdown]: Thanks

File: source/_integrations/manual_mqtt.markdown [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Manual MQTT
-description: Instructions on how to integrate manual alarms into Home Assistant with MQTT support.
+title: Manual MQTT Alarm
```
Comment (thanksmister) [inline source/_integrations/manual_mqtt.markdown:2]: Yes, name change desperately needed, thanks!

**Review by thanksmister — DISMISSED** [review]
Good documentation update, easy to read and understand.

**Review by thanksmister — APPROVED** [review]
Format and description for pending look good.

**Comment by derekatkins** [comment]
@klaasnicolaas I believe that I have resolved all your requested changes.  Can you please verify?
Thanks!

File: source/_integrations/manual_mqtt.markdown (hunk 1 above)
Comment (derekatkins) [reply source/_integrations/manual_mqtt.markdown]: I committed this change.  Thanks.

File: source/_integrations/manual_mqtt.markdown (hunk 2 above)
Comment (derekatkins) [reply source/_integrations/manual_mqtt.markdown]: I committed this change.  Thanks.

**Review by thanksmister — APPROVED** [review]

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**